mods/example/rules/palettes.yaml

```diff
@@ -0,0 +1,17 @@
+^palettes:
+	PaletteFromPng@cursor:
+		Name: cursor
+		Filename: bits/cursor.png
+		CursorPalette: true
+	PaletteFromPng@terrain:
+		Name: terrain
+		Filename: bits/terrain.png
+	PaletteFromPng@palette:
+		Name: palette
+		Filename: bits/palette.png
```

Comment:
Storing the palette in a JASC/Gimp palette text file might make more sense and show off an alternative way to set things up.